.scrutinizer.yml

```diff
@@ -32,7 +32,7 @@ build:
       tests:
         override:
           - command: |
-              ./yii serve 127.0.0.1:8080 > ./runtime/yii.log 2>&1 &
+              php -S 127.0.0.1:8080 -t public > ./runtime/yii.log 2>&1 &
```

Comment:
I'd leave it as is. Why changing it?

Reply:
It isn't Codeception problem either, I think that it depends on a version of xdebug.

Codeception's own code coverage test produces 3/3, 3/4, 4/4 and even 4/5 in different builds.
See https://github.com/Codeception/Codeception/pull/6439/files

Reply:
> It isn't Codeception problem either, I think that it depends on a version of xdebug.
> 
> Codeception's own code coverage test produces 3/3, 3/4, 4/4 and even 4/5 in different builds. See https://github.com/Codeception/Codeception/pull/6439/files

Which version of Xdebug works correctly ?

Reply:
Correctly or the same as before?

Probably the latest version is more correct, but you must do some work to get that 100% coverage again.

Reply:
Alright, it was Codeception issue, see https://github.com/Codeception/Codeception/pull/6448

Reply:
Work fine code coverage 100%